### DISCUSSION TITLE
Primeros Pasos

### DISCUSSION THREAD

**Discussion by ghiglionemag** [atralice/Curso.Prep.Henry#29] [General]
¡Hola! Quiero instalar el Node pero no encuentro una versión que funcione en Windows 7, ¿hay alguna otra versión que no haya encontrado que pueda instalar? ¡Saludos y gracias! :)

**Comment by ghiglionemag** [comment]
> ¡Hola! Quiero instalar el Node pero no encuentro una versión que funcione en Windows 7, ¿hay alguna otra versión que no haya encontrado que pueda instalar? ¡Saludos y gracias! :)

Hola, dejo por acá el link a la versión que sí funciono por sí alguien más la necesita https://nodejs.org/download/release/v13.14.0/node-v13.14.0-x64.msi :)

**Reply by awesomnet1** [reply]
Habría algún problema en el desarrollo de mis proyectos al usar una version de node tan antigua?.

**Comment by OlgaMG** [comment]
Hola, apenas iniciando con las instalaciones de software y no he podido instalar el node en windows 10 pro, porque me sale un error "A DLL required for this install to complete could not be run" y de ahí no hace nada mas, agradezco su apoyo, gracias

**Comment by waku1044** [comment]
Hola a todos , a mi me paso lo mismo con el Node.js , tengo win 7 32bit y no encunentro otro.

**Comment by yeniferparada** [comment]
al intetar sicronizar me sale este error, me podrian indicar que estoy haciendo mal
![image](https://user-images.githubusercontent.com/102178707/159594862-f21ec586-62a1-4a1f-95ea-e9233c5c294a.png)

**Reply by ronalcastro97** [reply]
me pasa lo mismo

**Comment by 2460-web** [comment]
Quiero ejecutar el comando Git commit -m "first commit" y no me deja, en su lugar me sale esto 
*** Please tell me who you are.

Run

  git config --global user.email "[you@example.com](mailto:you@example.com)"
  git config --global [user.name](http://user.name/) "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'gusty@Pc-de-Gusti.(none)')
PS C:\Users\gusty\OneDrive\Escritorio\Prep-Course> git omit
git: 'omit' is not a git command. See 'git --help'.

**Reply by ana12-pixel** [reply]
hola 
me pasa lo mismo alguien sabe como solucionarlo?

**Reply by Johannaresnick95** [reply]
Me pasa igual, por favor alguien que nos ayude...
Oops desde abril del año pasado estamos pidiendo ayuda y nada, que triste.  :(
